config/local_info.xml

```diff
@@ -37,10 +108,16 @@
         the 'scope' parameter will return all results regardless of scope, e.g.
         'get_site' would return all sites in all scopes.
         -->
+
         <default_scope>
             <name>EGI</name>
         </default_scope>
 
+        <!-- If set to true the initial state of the scope filter drop-downs on the
+             ngi,site,service, downtimes pages is selected for the default_scope
+        -->
+        <default_filter_by_scope>true</default_filter_by_scope>
```

Comment:
Shouldn't this be `false` to mimic existing behaviour?

Reply:
Yes, I believe so.

Reply:
```suggestion
        <default_filter_by_scope>false</default_filter_by_scope>
```

Reply:
Updated in latest commit - https://github.com/GOCDB/gocdb/blob/1b51b69234b9827a75535fc6fdd67d5a9c1006a9/config/local_info.xml#L150